### ISSUE TITLE
Week 6 announcements

### ISSUE THREAD

**Issue by joenash** [campus-experts/spring-2017#49]
## The TLDR this week:
**HUGE week of talks! Don't miss out on Lara Hogan or Phil Nash**

PLEASE NOTE: Lara's guest talk is being postponed and will not be on Wednesday 3rd, keep an eye out for the new date soon!
- [Week 6 todo](https://github.com/campus-experts/spring-2017/blob/master/todos/week-6.md)
- [Last week's video](https://www.dropbox.com/s/b1s37d9tbpmtrok/technottingham.mp4?dl=0), and last week's [tech talk](https://www.dropbox.com/s/gkxfg9sgb8vmxjz/jamesmilnertalk.mp4?dl=0).
- Complete this week's exercises, [write an abstract](https://github.com/campus-experts/spring-2017/blob/master/docs/write-an-abstract.md)
- This week we have two guest talks! [Phil Nash on Service Workers on 2nd May](https://github.com/campus-experts/spring-2017/issues/48), and Lara Hogan on Demystifying Public Speaking on 3rd May.

## Welcome to Week 6 🚩

Hey folks! Gentle reminder this week to [fill out the survey](https://goo.gl/forms/GWiihlMvYAmORs142). I'll also be [offline 4th-9th May](https://github.com/campus-experts/spring-2017/issues/43). 

This week is **huge** for guest talks. We're really lucky to be joined by [Lara Hogan](http://larahogan.me/). Lara is author of Demystifying Public Speaking, and she'll be giving us a talk on the same topic. Don't miss out! 

## This week on Slack
Missed a discussion on slack? Not to worry!
It's been a little quiet this week!

- Peter is speaking in week 8, [find out more about him](https://githubeducation.slack.com/archives/C4PF3VCSU/p1493292215045729).
- Slides for [James' talk](https://githubeducation.slack.com/archives/C2TQDRQSG/p1493312528014499).

**Comment by BlueHatbRit** [comment]
So excited for the talks this week!
